src/components/SidebarSubmenu.tsx

```diff
@@ -20,11 +20,15 @@ export function SidebarSubmenu({
   let slug = `${subpath}/${newLabel}`;
 
   useEffect(() => {
-    const pathArray = submenu![0].slug?.split('/');
-    const index = pathArray?.indexOf(subpath!);
-    const category = pathArray && index ? `/${pathArray[index + 1]}` : '';
-    const active = pathname?.startsWith(`/docs/${subpath}${category}/`);
-    setIsOpened(active);
+    if (pathname.includes('/guides/')) {
```

Comment:
the sidebar isn't getting `active` selection properly when inside guides pages

![CleanShot 2023-07-11 at 19 50 50@2x](https://github.com/FuelLabs/docs-hub/assets/2029172/b74426f0-41bc-49ae-af5d-cbc9a92c9888)